### PR TITLE
replace fluentd with promtail

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

**Review by theycallmemac — DISMISSED** [review]
Some of this is subject to change in the next few months but as of now this is good. Merging

**Comment by theycallmemac** [comment]
Actually, @butlerx why is the CI broken on this?

**Comment by theycallmemac** [comment]
Nvm just checked there, looks to be an exception cropping up during the pip install.

File: docs/index.md [hunk 1]
```diff
@@ -150,7 +149,6 @@ After each emergency remember to write a postmortem!
 
 Software written by RedBrick, or with RedBrick customisations etc.
 
-- [RedBrick Apt Repo (work in progress)](/procedures/redbrick-apt)
```
Comment (m1cr0man) [inline docs/index.md]: You need to actually delete that procedure md

File: docs/index.md [hunk 2]
```diff
@@ -176,4 +169,5 @@ accessed and read from anywhere.
 ### ReadTheDocs
 
 Docs are auto deployed to [readthedocs](https://readthedocs.io) on commit to the
-`master` branch. They should always be up to date.
+`master` branch. They should always be up to date. They are available at
+https://docs.redbrick.dcu.ie and https://redbrick.rtfd.io
```
Comment (m1cr0man) [inline docs/index.md]: Can we make these actual links? :)

**Review by m1cr0man — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (butlerx) [reply docs/index.md]: done

File: docs/index.md (hunk 1 above)
Comment (butlerx) [reply docs/index.md]: done

**Comment by butlerx** [comment]
@theycallmemac fixed tests

**Review by m1cr0man — APPROVED** [review]

**Pull request merged**